plugin/storage/es/factory.go

```diff
@@ -218,6 +218,7 @@ func createDependencyReader(
 		Client:              client,
 		Logger:              logger,
 		IndexPrefix:         cfg.GetIndexPrefix(),
+		IndexDateLayout:     cfg.GetIndexDateLayoutDependencies(),
```

Comment:
Thanks for fixing this, I wonder if we can update the unit tests to prevent this from happening again. Maybe via a type assertion of the returned `dependencystore.Reader`?

Reply:
not sure I find it useful, looks like a dummy test